sklearn/ensemble/_forest.py

```diff
@@ -2229,8 +2229,14 @@ class ExtraTreesRegressor(ForestRegressor):
 
     See Also
     --------
-    sklearn.tree.ExtraTreeRegressor : Base estimator for this ensemble.
+    ExtraTreesClassifier : An extra-trees classifier.
+    ExtraTreesRegressor : An extra-trees regressor.
+    RandomForestClassifier : A random forest classifier.
     RandomForestRegressor : Ensemble regressor using trees with optimal splits.
+    sklearn.tree.ExtraTreeClassifier: An extremely randomized
+        tree classifier.
+    sklearn.tree.ExtraTreeRegressor : An extremely randomized
+        tree regressor.
```

Comment:
You can remove these two. Basically, no one should use them in practice so there is no need to add a `See Also` for them.

Reply:
Done. Thank you for this insight.

Reply:
@glemaitre would you prefer if I remove these from other classifiers in ``_forest.py`` if I see them listed there?